templates/_base.html.twig

```diff
@@ -51,9 +51,16 @@
       {% block footer %}
         <p>
           Kind regards,<br>
-          The Big Give Team<br>
+          Big Give Team<br>
           www.thebiggive.org.uk<br>
         </p>
+
+        {# This check is added because we don't want this sentence to be included in the pledger-success template,
+        but we want it included in all other templates, which include the `donorFirstName` parameter. MAIL-17 #}
+        {% if donorFirstName %}
```

Comment:
@alihejazi If we're going to use a string var for this, I think we should use Twig's `is not empty` syntax so
* the behaviour is clear for new/returning devs, and
* we are not relying on type coercion and potentially introducing the surprises that can bring